### PR TITLE
Edited Ruby topic in openshift_sti_images as per NodeJS topic

### PR DESCRIPTION
**Pull request by tpoitras**
This text originally submitted in https://github.com/openshift/openshift-docs/pull/171 by @jhadvig 

Looks like an older version of the NodeJS topic (which I just cleaned up) was used as a template for this topic.

The content is good, but I've made some basic formatting and grammar edits. This made me look back at what I'd done with NodeJS, which was great, because I found a couple more things there that needed fixing.

### CODE REVIEW

**Comment by tpoitras** [comment]
Hey @jhadvig -- can you please do a tech review on this for me? It should all still be basically the same, I just cleaned it up a little. Thanks.

**Comment by adellape** [comment]
@tpoitras FYI, I merged @jhadvig's https://github.com/openshift/openshift-docs/pull/184 which reflects a refactor to the sti-ruby repo. That merge affects your PR here, so you'll need to rebase.

**Comment by tpoitras** [comment]
@adellape -- I rebased and had a conflict in the ruby.adoc file but I resolved it. Should be good now?

File: openshift_sti_images/ruby.adoc [hunk 1]
```diff
@@ -92,43 +91,90 @@ $ make test VERSION=2.0
...
+****
+
+Where:
 
-. Platform_name - Refers to the STI platform; `ruby` is one example
-. Platform_version - The version of the referenced platform, without dots (like `20` for Ruby 2.0)
-. Base_builder_image - The base OS, like `rhel7` or `centos7`
+. Platform_name - Refers to the STI platform; for example, `ruby`
+. Platform_version - The version of the referenced platform, without dots; for example, `20` for Ruby 2.0
+. Base_builder_image - The base OS, such as `rhel7` or `centos7`
```
Comment (adellape) [inline openshift_sti_images/ruby.adoc]: Mark up "platform_name", etc in this list?

File: openshift_sti_images/ruby.adoc [hunk 2]
```diff
@@ -92,43 +91,90 @@ $ make test VERSION=2.0
...
-* openshift/ruby-20-rhel7
+openshift/ruby-20-rhel7
+----
+====
 
 == Environment variables
 
-* *RACK_ENV* - This variable specifies in what environment should the ruby application be deployed - `production`, `development`, `test`. Each level has different behavior in terms of logging verbosity, error pages, ruby gem installation, etc.
-** *Note* : The application assets are going to be compiled only if the RACK_ENV is set to production
+* [envvar]#RACK_ENV# - This variable specifies the environment within which the Ruby application is deployed; for example, `production`, `development`, or `test`. Each level has different behavior in terms of logging verbosity, error pages, and ruby gem installation. The application assets are only compiled if [envvar]#RACK_ENV# is set to production.
```
Comment (adellape) [inline openshift_sti_images/ruby.adoc]: "production" in the last sentence can be marked up.

File: openshift_sti_images/ruby.adoc [hunk 3]
```diff
@@ -92,43 +91,90 @@ $ make test VERSION=2.0
...
 
-* *RAILS_ENV* - This variable specifies in what environment should the Ruby on Rails application be deployed - `production`, `development`, `test`. Each level has different behavior in terms of logging verbosity, error pages, ruby gem installation, etc.
-** *Note* : The application assets are going to be compiled only if the RAILS_ENV is set to production
+* [envvar]#RAILS_ENV# - This variable specifies the environment within which the Ruby on Rails application is deployed; for example, `production`, `development`, or `test`. Each level has different behavior in terms of logging verbosity, error pages, and ruby gem installation. The application assets are only compiled if [envvar]#RAILS_ENV# is set to production.
```
Comment (adellape) [inline openshift_sti_images/ruby.adoc]: Same here re: "production".

**Comment by adellape** [comment]
@tpoitras For both of these files, wondering if all headings should be title case.

Will wait for tech ack from @jhadvig before merging.

**Comment by tpoitras** [comment]
Thanks, @adellape -- I've made those changes.

Waiting on teck ack from @jhadvig

File: openshift_sti_images/ruby.adoc [hunk 4]
```diff
@@ -48,42 +46,43 @@ $ make build VERSION=2.0
...
 ----
 $ curl 127.0.0.1:9292
 ----
 
 == Test
-OpenShift https://github.com/openshift/sti-ruby/tree/master/2.0[STI-Ruby] repository also provides STI test framework, which launches test to check functionality of a simple ruby application built on top of sti-ruby image.
+The OpenShift https://github.com/openshift/sti-ruby/tree/master/2.0[STI-Ruby] repository also provides the STI test framework, which tests the functionality of a simple Ruby application built on top of STI-Ruby image.
 
-=== RHEL-7 base image
+=== RHEL7 Base Image
```
Comment (jhadvig) [inline openshift_sti_images/ruby.adoc]: Missing space between disto name and version

**Comment by jhadvig** [comment]
@tpoitras @adellape just a small nit.
Other then that LGTM :+1:

**Comment by adellape** [comment]
Going to merge this for now and we can get those headings in a follow-up.

**Pull request merged**